src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1932,6 +1952,28 @@ public EntryResult tManufacturedProduct2Medication(ManufacturedProduct cdaManufa
 			return result;
 
 		Medication fhirMedication = new Medication();
+
```

Comment:
This was further down in the function and was only used to add the medication code. However, I added the dedup step here, so I moved it to the front.